### PR TITLE
fix: client getter for nft with accountId

### PR DESCRIPTION
**Pull request by bradleySuira**
Added accountId to client getter inside NFT send transaction

### CODE REVIEW

File: packages/wallet-core/src/store/actions/sendNFTTransaction.ts [hunk 1]
```diff
@@ -12,7 +12,9 @@ export const sendNFTTransaction = async (
   const { getters, commit, dispatch } = rootActionContext(context);
   const account: Account = getters.accountItem(accountId)!;
   const asset = getChain(network, account.chain).nativeAsset[0].code;
-  const client = getters.client({ network, walletId, chainId: account.chain });
+  const client = getters.client({ network, walletId, chainId: account.chain, accountId });
+  // eslint-disable-next-line no-debugger
+  debugger
```
Comment (monokh) [inline packages/wallet-core/src/store/actions/sendNFTTransaction.ts]: debugger line ?

**Review by monokh — CHANGES_REQUESTED** [review]

File: packages/wallet-core/src/store/actions/sendNFTTransaction.ts (hunk 1 above)
Comment (bradleySuira) [reply packages/wallet-core/src/store/actions/sendNFTTransaction.ts]: fixed now

File: .changeset/sour-spies-repair.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@liquality/wallet-core': major
```
Comment (monokh) [inline .changeset/sour-spies-repair.md]: should be patch?

Comment (bradleySuira) [reply .changeset/sour-spies-repair.md]: updated

**Review by monokh — APPROVED** [review]

**Pull request merged**